src/assets/js/api-service.js

```diff
@@ -4,18 +4,39 @@ const AWS_IRMA_BACKEND_SERVER = 'https://f9emnttxd6.execute-api.ap-southeast-2.a
 
 const SECOND_PRIVATE_TOKEN = 'slU2raoR4f6q1eiVQbYuk5GGI3j2qaL25BdB6SSA';
 
-const checkPassport = (base64Image) => {
-  return _apiSingleSourcePost('passports', {
-    document_image: base64Image,
-    country_code: 'NZL',
-  });
+const checkPassport = (base64Image, nameArray) => {
+  familyName = nameArray.pop();
```

Comment:
This does not work for the family names which have two parts, like mine! I think the first element of the list should be the first name and the rest of the list last name.

Reply:
True! But changing it will mean it stops working with middle names. The only way to accurately use the name would be to change the issuer to issue family name and given names separately, there's no way to programmatically split the full name in a way that works in every case without more information.

Reply:
You are right about that. But here we assume we just have First and Last names. Actually we are doing the same to show First Name here: https://github.com/InternetNZ/irma-demo-ui/blob/2037c543cd1470a7d60a87fa6420de9ab3718f70/src/id-card/index.html#L384
So better to be consistent.